draft-js-mention-plugin/src/utils/getSearchText.js

```diff
@@ -1,12 +1,10 @@
-import getWordAt from './getWordAt';
```

Comment:
have you removed getWordAt, now that you've replaced it? I can't see the deleted file in the diff...